be/src/formats/orc/apache-orc/c++/src/sargs/SearchArgument.cc

```diff
@@ -197,8 +197,9 @@ SearchArgumentBuilder& SearchArgumentBuilderImpl::addChildForIn(T column, Predic
     return *this;
 }
 
-SearchArgumentBuilder& SearchArgumentBuilderImpl::in(const std::string& column, PredicateDataType type,
-                                                     const std::vector<Literal>& literals) {
+template <typename T>
+SearchArgumentBuilder& SearchArgumentBuilderImpl::addChildForIn(T column, PredicateDataType type,
```

Comment:
you can check if apache/orc already has this piece of code. `https://github.com/apache/orc/`

Reply:
https://github.com/apache/orc/